modules/ray_tracing/src/userobjects/RayTracingStudy.C

```diff
@@ -1253,11 +1247,6 @@ RayTracingStudy::rayBank() const
 std::shared_ptr<Ray>
 RayTracingStudy::getBankedRay(const RayID ray_id) const
 {
-  if (!_bank_rays_on_completion)
-    mooseError("Cannot get a banked Ray with _bank_rays_on_completion = false.");
-  if (currentlyGenerating() || currentlyPropagating())
-    mooseError("Cannot get the Ray bank during generation or propagation.");
-
   // This is only a linear search - can be improved on with a map in the future
```

Comment:
this was meant to protect against the wrong use of this API right
why not an assert?

Reply:
This check was a duplicate and is already checked in `rayBank()` (and tested for)

Reply:
and rayBank is indeed called in this routine already. all good!